unity-renderer/Assets/Scripts/MainScripts/DCL/Components/TextShape/TextShape.cs

```diff
@@ -82,14 +82,6 @@ private void OnUpdate()
             {
                 transform.forward = mainCamera.transform.forward;
             }
-            
-            var opacity = cachedModel.visible ? cachedModel.opacity : 0;
-            var totalSize = cachedModel.fontSize;
-            var totalScale = text.transform.lossyScale.sqrMagnitude;
-            bool isVisible = opacity > 0 && totalSize > 0 && totalScale > 0 && entity.isInsideSceneBoundaries;
```

Comment:
I think we should be aware that just a little value of 0.0001 or -0.0001 in any of scale components of any parent of text will make totalScale > 0 equal true. Maybe there should be a threshold a little different from 0.

Reply:
Agreed with Anton! If we bring this optimization back, we should add some delta threshold to determine the visibility of the TextShape.

Reply:
don't think we need to bring the optimization back, if you look at the issue I've posted the benchmark and in ice poker (one of the places with most intense usage of text shapes) there is actually not that big of an impact